yjit/src/codegen.rs

```diff
@@ -4310,6 +4357,8 @@ fn gen_opt_newarray_send(
         gen_opt_newarray_max(jit, asm)
     } else if method == VM_OPT_NEWARRAY_SEND_HASH {
         gen_opt_newarray_hash(jit, asm)
+    } else if method == VM_OPT_NEWARRAY_SEND_INCLUDE_P {
+        gen_opt_newarray_include_p(jit, asm)
     } else if method == VM_OPT_NEWARRAY_SEND_PACK {
         gen_opt_newarray_pack_buffer(jit, asm, 1, None)
     } else if method == VM_OPT_NEWARRAY_SEND_PACK_BUFFER {
```

Comment:
Are all these functions mostly doing the same thing, and if so, could we make them generic on the function being called? I know this is not your doing. Just wondering if we can avoid doing boilerplate for every newarray send type.

Reply:
Sounds good 👍